code/modules/hydroponics/grown/towercap.dm

```diff
@@ -238,3 +268,7 @@
 /obj/structure/bonfire/unbuckle_mob(mob/living/buckled_mob, force = FALSE)
 	if(..())
 		buckled_mob.pixel_y -= 13
+
+/obj/structure/bonfire/examine(mob/user)
+	. = ..()
+	. += "<span class='notice'>It looks like it'll burn for [time_left()] seconds and has around [round(fuel)] units of fuel left, you can add more fuel by adding logs or planks of wood to [src].</span>"
```

Comment:
code is fine, maybe this would be a good candidate for the more detailed examine proc?

Reply:
It looks nice when view in game and doesn't look bloated. You're already inspecting it once, doing it again seems redundant <3

Reply:
sounds good